### PR TITLE
Update install criteria doc

### PR DESCRIPTION
**Pull request by petele**
Changes proposed in this pull request:
- Updates to the [Improving Progressive Web App offline support detection](https://developer.chrome.com/blog/improved-pwa-offline-detection/) article for current plans

### CODE REVIEW

**Comment by petele** [comment]
cc: @b1tr0t

File: site/en/blog/improved-pwa-offline-detection/index.md [hunk 1]
```diff
@@ -3,26 +3,26 @@ layout: 'layouts/blog-post.njk'
...
-**TL;DR:** Offline support has been part of the PWA installability criteria
-since the beginning. We are updating the offline detection logic to ensure a
-PWA actually provides an offline experience, closing the loophole used by
-some developers to meet the installability criteria. In the future,
-sites with empty `fetch` event handlers will no longer meet the criteria.
+{% Aside 'note' %}
+**Updated April 14th, 2021:** We previously announced plans to update the
+installability criteria to ensure a PWA actually provides an offline experience.
+After listening to your feedback, and discovering a number of issues, **we have
+decided to put those plans on hold**. We strongly believe providing a valid
```
Comment (jpmedley) [inline site/en/blog/improved-pwa-offline-detection/index.md:24]: This is not a blocking request. Typography is not a substitute for good writing. You've already emphasized this by making it a note. Why not open with "Plans for updating installability criteria are on hold."

**Review by jpmedley — APPROVED** [review]
I'm approving this, but I left some food for thought about one of your notes. Please consider it.

**Review by jpmedley — APPROVED** [review]
